### PR TITLE
Add blog on Custom DNS entries

### PR DESCRIPTION
**Pull request by johnbelamaric**

### CODE REVIEW

File: content/post/custom-dns-and-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+IP address for the service. Inside the cluster, it will resolve to the same thing, and so using this name internally
+will cause traffic to hairpin - travel out of the cluster and then back in via the external IP. Instead, we want it
+to resolve to the internal ClusterIP, avoiding the hairpin.
+
+To do this in CoreDNS, we make use of the `rewrite` middleware. This middleware can modify a query before it is sent
+down the chain to whatever backend is going to answer it. Recall the `Corefile` (CoreDNS configuration file) we
+used in the last blog:
+
+~~~ txt
+    .:53 {
```
Comment (miekg) [inline content/post/custom-dns-and-kubernetes.md]: nit: doesn't need indentation

File: content/post/custom-dns-and-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        cache 30
+    }
+~~~
+
+Once we add that to the `ConfigMap` via `kubectl edit` or `kubectl apply`, we have to let CoreDNS know that the `Corefile`
+has changed. You can send it a `SIGUSR1` to tell it to reload graceful - that is, without loss of service:
+
+~~~ txt
+$ kubectl exec -n kube-system coredns-980047985-g2748 -- kill -SIGUSR1 1
+$
```
Comment (miekg) [inline content/post/custom-dns-and-kubernetes.md]: remove second `$`

**Comment by miekg** [comment]
lgmt!

How do you want to pace these? We have an upcoming release (so 3 blogs in the pipeline)

**Comment by johnbelamaric** [comment]
Good question. This one specifically mentions that it is before the 007 release, but I can change that. I just didn't want to make it super long and would rather get one more blog out of it 😄 .

I can change it to say "007 was recently released" and then we can still have a follow on instead.

The minikube one I just wrote because I found the issue while writing the first one. Maybe we can put that one out now or along with the 007 blog, then tweak this one and put it out a week later?

**Pull request merged**